src/Util/AdminObjectAclManipulator.php

```diff
@@ -195,11 +186,9 @@ protected function buildAcl(AdminObjectAclData $data, FormInterface $form, \Trav
     /**
```

Comment:
The method `protected function getSecurityIdentity($aclValue)` could return SecurityIdentityInterface instead of the current return type.